### ISSUE TITLE
Cluster pods fail at bootstrap checks

### ISSUE THREAD

**Issue by Overbryd** [upmc-enterprises/elasticsearch-operator#138]
Hi,

another thing I found deployment an example elasticsearch cluster on Kubernetes v1.8 on the Google Compute Engine.

The pods error out with this particular in the logs:

```
ERROR: [1] bootstrap checks failed
[1]: max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
```

Maybe something for the operator, so it might want to the check that the target instances support this sysconfig setting?

Any experience with running this on GCE?

**Comment by Overbryd** [comment]
I found an issue in the pires/kubernetes-elasticsearch-cluster project. https://github.com/pires/kubernetes-elasticsearch-cluster/issues/85

Adding the following the my operator manifest configures all nodes properly:

```yaml
---
apiVersion: extensions/v1beta1
kind: DaemonSet
metadata:
  name: es-sysctl
  namespace: elasticsearch
spec:
  selector:
    matchLabels:
      name: es-sysctl
  template:
    metadata:
      labels:
        name: es-sysctl
    spec:
      containers:
      - name: es-sysctl
        image: gcr.io/google-containers/startup-script:v1
        imagePullPolicy: IfNotPresent
        securityContext:
          privileged: true
        env:
        - name: STARTUP_SCRIPT
          value: |
            #!/bin/sh
            sysctl -w vm.max_map_count=262144
            echo "done"
```

**Comment by stevesloka** [comment]
Hey @Overbryd I haven't run into this issue yet, but might be good to add the DS so the operator creates for you. Thanks for the heads up!

**Comment by stevesloka** [comment]
Fixed in #149

**Comment by Shifter2600** [comment]
The namespace is wrong it needed to be operator in the apiVersion: extensions/v1beta1 above.
Also Thx this worked. 
I was able to get my cluster going now using rook.
I can confirm the example controller with this script added and the rook example is working on Rancher 2.0 tech preview on Kubernetes 1.9
